pandas/core/indexes/datetimes.py

```diff
@@ -1057,7 +1057,30 @@ def to_pydatetime(self):
 
     def to_period(self, freq=None):
         """
-        Cast to PeriodIndex at a particular frequency
+        Cast to PeriodIndex at a particular frequency.
+
+        Converts timestamps index to period.
+
+        Parameters
+        ----------
+        freq : string or pandas offset object,optional
+            One of pandas date offset string or corresponding objects.
```

Comment:
Could you include a link to the table of alias. Something like

    :ref:`offset aliases <timeseries.offset_aliases>`